experiments/run_experiment.py

```diff
@@ -214,59 +232,153 @@ def _get_mrmc(
         rescale_direction=mrmc_method.get_constant_step_size_rescaler(
             step_size
         ),
-        random_seed=random_seed,
-        confidence_threshold=confidence_threshold,
+        confidence_threshold=confidence_cutoff,
```

Comment:
For consistency, it would be better to use 'threshold' everywhere or 'cutoff' everywhere, for this parameter as well as any others (e.g. volcano_cutoff <--> volcano_threshold)

Reply:
Yes! Agreed, this has been bothering me. The terminology is used interchangeably throughout the repo (I think there's `confidence_cutoff`, `confidence_threshold`, `desired_confidence`, and `target_confidence`). Maybe since it's a large (ie many-file) change, I should do it in a separate PR?